app/views/time_slot/index.html.erb

```diff
@@ -7,7 +7,7 @@
       </h6>
     </div>
   <% end %>
-  <% if @appointment.present? && @appointment.start > Time.zone.now %>
```

Comment:
This was causing some edge cases where I would have an appointment in the past, see "Voce nao tem agendamentos" but then see a "Substituir" in order to replace an appointment.

Although I'm not sure what the flow is in case someone loses an appointment (scheduled but never checked in).

Reply:
hummmm good question I think a patient is not supposed to see the appointment if the appointment is in the past. But I'm not sure why :sweat_smile: Maybe we can open a issue to sort this out

Reply:
I think we can show a "Past appointments" with no option to cancel/replace it. I disagree that we should hide it all together.

Reply:
As a patient, given that we provide no receipt for the transaction, it would be to the very least bothersome that I can't see a past appointment if it wasn't attended, as we are giving the option for the patient to re-schedule but not to see what the missed appointment was about.

Reply:
Agreed! 

WDYT about making this change on a separate PR? Therefore this PR will be a enhancements only and we can refine a little more this solution you proposed.

Reply:
I'll work on showing past appointments in another PR :-) No need to make this one even longer.